egs/iwslt22_ta/ST/local/prepare_transcripts.py

```
@@ -1,66 +1,66 @@
-        for c in cuts:
-            src_txt = c.supervisions[0].text
-            tgt_txt = c.supervisions[0].custom['translated_text']['eng']
-            src.write(src_txt + '\n')
-            tgt.write(tgt_txt + '\n')
-
-if __name__ == "__main__":
-    main()
\ No newline at end of file
+# Copyright 2023 Johns Hopkins University  (Amir Hussein)
+
+#!/usr/bin/python
```

Ruff (0.14.13)

Shebang should be at the beginning of the file

(EXE005)